pkg/engine/backend/init/init.go

```diff
@@ -0,0 +1,21 @@
+package init
+
+import (
+	"kusionstack.io/kusion/pkg/engine/states"
+	"kusionstack.io/kusion/pkg/engine/states/local"
+)
+
+// backends store all available backend
+var backends map[string]func() states.Backend
+
+// BackendInit init backends map with all support backend
+func BackendInit() {
```

Comment:
kusion use file_system_state as default backend, how about defualt it as `init()`, so that `kusion apply` only focus on input arguments when passed `backend-type` or `backend-config`